public/js/bio-link/share.js

```
@@ -0,0 +1,140 @@
+function shareToEmail() {
+    const url = document.getElementById('shareBioLinkUrl').value;
+    const subject = 'Check out my bio link';
+    const body = `I'd like to share my bio link with you: ${url}`;
+    window.location.href = `mailto:?subject=${encodeURIComponent(subject)}&body=${encodeURIComponent(body)}`;
+}
+
+// ================================
+// NATIVE SHARE API
+// ================================
+
+async function nativeShare() {
```

ESLint

[error] 'nativeShare' is defined but never used.

(no-unused-vars)